BootloaderCommonPkg/Library/ShellLib/CmdBoot.c

```diff
@@ -369,9 +367,6 @@ PrintBootOption (
                  );
     }
 
-    if (Index == mCurrentBoot) {
-      ShellPrint (L" *Current");
-    }
```

Comment:
I think it is good to print out a marker for current boot option. Why do we remove this ?

Reply:
I was even thinking maybe we could add this to the OS Loader boot option printing, it made it easier to know which boot option would be tried next.

Reply:
I agree this is good to have. Next PRs would handle this. Let me keep the mCurrentBoot variable in this commit as a initial patch. Eventually, this global variable needs to be removed and we need to improve GetCurrentBootOption() and GetNextBootOption() more intuitive (maybe adding SetCurrentBootOption as well).